src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/repository/BankHolidayCacheRepository.kt

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.digital.hmpps.calculatereleasedatesapi.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import uk.gov.justice.digital.hmpps.calculatereleasedatesapi.entity.BankHolidayCache
+
+@Repository
+interface BankHolidayCacheRepository : JpaRepository<BankHolidayCache, Long> {
+
+  fun findFirstByOrderByIdAsc(): BankHolidayCache?
```

Comment:
will this get the first matching bank holiday when ordered by ascending? In which case shouldn't it order in  Descending?

Reply:
So there is only ever one entry in the database. The schedule will just update the row every night as there is no need to capture historic bank holiday data.